Description: Explicitly exclude /unknownerror from the SAFERERRORLIST
 Since we remove /unknownerror from errordict,
 we want to exclude it
 from the list we copy to gserrordict for SAFERERRORS
Origin: upstream, http://git.ghostscript.com/?p=ghostpdl.git;h=f1a0787
Author: Chris Liddell <chris.liddell@artifex.com>
Forwarded: yes
Last-Update: 2018-10-18
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Resource/Init/gs_init.ps
+++ b/Resource/Init/gs_init.ps
@@ -1160,7 +1160,13 @@
 % Put all the requested handlers in gserrordict
   gserrordict
   //.SAFERERRORLIST
-  {dup errordict exch get 2 index 3 1 roll put} forall
+  {
+    dup /unknownerror eq
+    {pop}
+    {
+      dup errordict exch get 2 index 3 1 roll put
+    }ifelse
+  }forall
   noaccess pop
   systemdict /.setsafeerrors .forceundef
   systemdict /.SAFERERRORLIST .forceundef
